oak/server/rust/oak_runtime/src/node/grpc/client.rs

```diff
@@ -57,6 +59,22 @@ fn check_uri(uri: &Uri) -> Result<(), ConfigurationError> {
         .ok_or(ConfigurationError::NoHostElement)
 }
 
+fn grpc_client_node_privilege(uri: &Uri) -> NodePrivilege {
+    if uri.scheme() == Some(&http::uri::Scheme::HTTPS) {
+        // Authority is the host:port portion of the endpoint name.
+        if let Some(authority) = uri.authority() {
+            NodePrivilege::new(
+                hashset! { oak_abi::label::tls_endpoint_tag(&authority.as_str()) },
+                hashset! { oak_abi::label::tls_endpoint_tag(&authority.as_str()) },
```

Comment:
Why is this string copied two times?

Reply:
One is the privilege to declassify, and the other is the privilege to endorse. Basically confidentiality + integrity. In this case (and in many cases in practice) they happen to coincide.

Reply:
Should we use different types so it will be harder to make mistakes when setting labels?
Or confidentiality and integrity are always the same?

Reply:
Maybe I should just remove the constructor, and build the struct directly with the field names, so at least the values are next to their description.

They are of the same type, so I don't think we can use different types though.